app/controllers/product_review_videos/upload_contexts_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ProductReviewVideos::UploadContextsController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    render json: {
+      aws_access_key_id: AWS_ACCESS_KEY,
+      s3_url: "https://s3.amazonaws.com/#{S3_BUCKET}",
+      user_id: logged_in_user.external_id,
+    }
```

Comment:
So that we can fetch this async and don't have to add this to all the places where a review form is shown.

Follows (for example)

https://github.com/antiwork/gumroad/blob/4938009a90560339eac5c2e9b7624b22165e7573/app/presenters/installment_presenter.rb#L135-L137